activities/procurement/principles.md

```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
 * sustainable for the organization as well as society and the planet
 
 The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+
+## Procurement process
+
+To make sure the procurement process is fair, we adhere to the following requirements:
+
+* we request 3 - 4 quotes from different providers
```

Comment:
I'd keep this at 'a proposal' from one party, unless we hit the European limits, at which moment we can go with a more open procurement.